src/asm/aarch64/transform/mod.rs

```diff
@@ -7,4 +7,4 @@
 // Media Patent License 1.0 was not distributed with this source code in the
 // PATENTS file, you can obtain it at www.aomedia.org/license/patent.
 
-pub mod inverse;
+//pub mod inverse;
```

Comment:
Why this?

Reply:
I missed it when reenabling the forward and inverse transforms.